public/app/plugins/datasource/opentsdb/components/OpenTsdbDetails.tsx

```diff
@@ -53,6 +53,30 @@ export const OpenTsdbDetails = (props: Props) => {
           onChange={onInputChangeHandler('lookupLimit', value, onChange)}
         />
       </div>
+      <div className="gf-form-inline">
+        <div className="gf-form">
+          <FormField
+            labelWidth={7}
+            label="Min time interval"
+            inputEl={
+              <Input
+                className={'width-6'}
+                value={value.jsonData.interval || ''}
+                onChange={onInputChangeHandler('interval', value, onChange)}
+                placeholder="1m"
```

Comment:
Having a placeholder for 1m but then defaulting to 30s seems misleading. We should probably leave it blank (if we won't have any default, see my other comment) and just explain the values in a tooltip.